views/newPatient.ejs

```diff
@@ -46,17 +46,17 @@
       <div>
         <div class="rf-my-3w">
           <!--todo : little red star for required fields -->
-          <label class="rf-label" for="firstnames">Prénoms</label>
-          <input class="rf-input midlength-input" type="text" id="firstnames" name="firstnames" required>
+          <label class="rf-label" for="firstNames">Prénoms</label>
```

Comment:
le HTML parfois ca met tout en lowercase, du coup pour eviter les bugs c'est plutot mieux d'utiliser du lowercase expres. 
"id" et "name" on peut mettre des majuscules, par contre d'autres attributes parfois non, et "for" j'arrive pas à trouver une reference.
Dans le doute pour eviter les bugs j'aime bien tout mettre en minuscules (ici, si le "for" est pété à cause des majuscules, on va peter les screenreaders)

(mais du coup attention au passage en JS, où on utilise du camelCase ! `const lastName = req.body['lastname']`)

Edit : meme commentaire pour tous les champs : firstname, lastname, ine

Reply:
bien compris